test/ng/compileSpec.js

```diff
@@ -11935,13 +11935,13 @@ describe('$compile', function() {
       testReplaceElementCleanup({});
     });
     it('should clean data of elements removed for directive templateUrl', function() {
-      testReplaceElementCleanup({asyncTmeplate: true});
+      testReplaceElementCleanup({asyncTemplate: true});
```

Comment:
Good catch. Since the tests pass, the intended behavior has not been broken sometime in the past.